lib/drupal.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
   };
 
   module.majorVersion = function() {
-    var version = grunt.config('drupal.drupal-version') || '',
+    var version = process.env.GDT_DRUPAL_CORE || grunt.config('drupal.drupal-version') || '',
```

Comment:
I couldn't actually find anywhere else where we are needing the version (drush usually handles that). We could alternatively use drush here to determine the version (as a 3rd fallback)?